deployment/quick-start/services-configuration.yaml.template

```diff
@@ -137,3 +137,7 @@ pylon:
   port: 80
   # deploy service for cloud load balancer
   deploy_service: false
+
+etcd-for-service:
+  use_etcd: false
```

Comment:
What‘s this label for?

Reply:
if it is true, rest-server will use this etcd instance as datastore
if it is false,  rest-server will use pai's k8s etcd as datastore (previous solution)

Reply:
No matter the cluster is deployed by pai or others, I think store data on the external etcd will be better. Using the etcd inside k8s is not a good choice. Let's correct this

Reply:
agree, as aks pr haven't mergeed and integrated, for a more robust transition. How about after end to end aks pr is merged, i will remove this switch.